### DISCUSSION TITLE
Docker mayhem.

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#463] [General]
VWsFriend docker image repository is likely to move away from docker hub after Docker announced even open source has to pay.

https://blog.alexellis.io/docker-is-deleting-open-source-images/

Any ideas yet?

**Comment by tillsteinbach** [comment]
I have always been pushing to github repository already: https://github.com/tillsteinbach?tab=packages&repo_name=VWsFriend
But as I use a private docker account and not a team account I currently see no issue with docker hub also.
